backend/learnings/tests.py

```diff
@@ -1,3 +1,59 @@
-from django.test import TestCase
+from django.contrib.auth.models import User
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APITestCase
 
-# Create your tests here.
+from .models import Category
+
+
+class CategoryAPIViewTests(APITestCase):
+    add_category_url = "http://127.0.0.1:8000/add_category/"
+    category_list_url = "http://127.0.0.1:8000/category_list/"
+    category_per_user_url = "http://127.0.0.1:8000/category/"
+    delete_category_url = "http://127.0.0.1:8000/delete_category/"
+    edit_category_url = "http://127.0.0.1:8000/edit_category/"
```

Comment:
please change to reverse instead of manually writing the urls
you can check out the documentation here :bow: 
https://docs.djangoproject.com/en/4.1/ref/urlresolvers/